tests/ask.bats

```diff
@@ -41,51 +43,52 @@ setup() {
   assert_failure
 }
 
+# This test simulates user input for ARGO_HOST and KUBEFLOW_HOST parameters
 @test "ask env: should print messages if parameters need to set for different component discriminators" {
-  run  ask env "ARGO_HOST" -m "parameter ingress.hosts test1" +empty -d ".ask.env" -d ".env" -ask-env
+  run ask env "ARGO_HOST" -m "parameter ingress.hosts test1" +empty -d ".env" -ask-env <<<"test_value_for_ARGO_HOST"
```

Comment:
Instead of `".env"` better to use `DOTENV_FILE` env var. Same in other test cases.